root/statistics/Index.js

```diff
@@ -1353,6 +1353,22 @@ component Index(
             <td>{fc('vote.abstain')}</td>
             <td>{fp('vote.abstain', 'vote')}</td>
           </tr>
+          <tr>
+            <th />
+            <th colSpan="3">
+              {addColonText(lp_statistics('Admin approval', 'vote'))}
```

Comment:
Can't these use the strings from mb_server instead?

Reply:
tbh I'm not sure? I assume if it could we would do it already for the other votes...